build-system/tasks/visual-diff/index.js

```diff
@@ -107,8 +105,39 @@ const SNAPSHOT_ERROR_SNIPPET = fs.readFileSync(
   'utf8'
 );
 
+// Browser instance that runs all visual tests.
 let browser_;
 
+/**
+ * @typedef {{
+ *  name: string,
+ *  message: string,
+ *  error: Error,
+ *  consoleMessages: puppeteer.ConsoleMessage[],
+ * }}
+ */
+let TestErrorDef;
+
+/**
+ * @typedef {{
+ *  url: string,
+ *  name: string,
+ *  viewport: {
+ *    width: number,
+ *    height: number,
+ *  },
+ *  loading_incomplete_selectors: string[],
+ *  loading_complete_selectors: string[],
+ *  loading_complete_delay_ms: number,
+ *  enable_percy_javascript: boolean,
+ *  interactive_tests: string,
+ *  no_base_test: boolean,
+ *  flaky: boolean,
+ *  tests_: Object<string,Function>,
```

Comment:
Supernit: should there be a space? `Object<string, Function>`

Reply:
I like it! Fixed.